iml-warp-drive/src/cache.rs

```diff
@@ -339,33 +339,24 @@ pub async fn populate_from_db(
 
     tracing::debug!("Built initial db statements");
 
-    let fut1 = future::try_join5(
-        into_row(client.query_raw(&stmts[0], iter::empty()).await?),
-        into_row(client.query_raw(&stmts[1], iter::empty()).await?),
-        into_row(client.query_raw(&stmts[2], iter::empty()).await?),
-        into_row(client.query_raw(&stmts[3], iter::empty()).await?),
-        into_row(client.query_raw(&stmts[4], iter::empty()).await?),
-    );
+    let managed_target_mount = into_row(client.query_raw(&stmts[0], iter::empty()).await?).await?;
+    let stratagem_configuration =
+        into_row(client.query_raw(&stmts[1], iter::empty()).await?).await?;
+    let lnet_configuration = into_row(client.query_raw(&stmts[2], iter::empty()).await?).await?;
+    let volume = into_row(client.query_raw(&stmts[3], iter::empty()).await?).await?;
+    let volume_node = into_row(client.query_raw(&stmts[4], iter::empty()).await?).await?;
```

Comment:
I was thinking we could keep the `fut{1,2,3}` as they were and just run them serially to keep some parallelism instead of running everything serially. But since this only runs on startup it should be ok.